_posts/2020-03-05-grayskull.md

```diff
@@ -0,0 +1,331 @@
+  sha256: 0d5fe9189a148acc3c3eb2ac8e1ac0742cb7618c084f3d228baaec0c254b318d
+
+build:
+  skip: True  # [py27]
+  number: 1
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  entry_points:
+    - py.test = py.test:main
+    - pytest = py.test:main
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=40.0
+    - setuptools_scm
+  run:
+    - atomicwrites >=1.0  # [win]
+    - attrs >=17.4.0
+    - colorama  # [win]
+    - importlib_metadata >=0.12  # [py<38]
+    - more-itertools >=4.0
+    - packaging
+    - pathlib2 >=2.2.0  # [py<36]
+    - pluggy >=0.12,<1.0
+    - py >=1.5.0
+    - python
+    - setuptools >=40.0
+    - wcwidth
+  run_constrained:
+    # pytest-faulthandler 2 is a dummy package.
+    # if an older version of fault-handler is installed, it will conflict with pytest >=5.
+    - pytest-faulthandler >=2
+
+test:
+  commands:
+    - pytest -h
+  imports:
+    - pytest
+
+about:
+  home: https://docs.pytest.org/en/latest/
+  license: MIT
+  license_file: LICENSE
+  summary: 'Simple and powerful testing with Python.'
+  description: |
+    The pytest framework makes it easy to write small tests, yet scales to
+    support complex functional testing for applications and libraries.
+  doc_url: https://docs.pytest.org/en/latest/
+  dev_url: https://github.com/pytest-dev/pytest/
+
+extra:
+  recipe-maintainers:
+    - flub
+    - goanpeca
+    - nicoddemus
+    - ocefpaf
+    - mingwandroid
+```
+
+#### Major differences
+
+|                   | Grayskull (0.1.9)     | Skeleton (3.18.11)         |
+|------------------:|:----------------------:|:---------------------------:|
+| **Command**       | <span style="color: green;">grayskull pypi pytest</span> | <span style="color: green;">conda skeleton pypi pytest</span> |
+| **Time**          | <span style="color: green;">4 seconds</span>            | <span style="color: red;">31 seconds</span>                 |
+| **License**       | <span style="color: green;">Added the license file and license type correctly</span> | <span style="color: brown;">Added just the license type</span> |
+| **Host Requirements** | <span style="color: green;">Added correctly all the host requirements</span> | <span style="color: red;">it didn't add the correctly the host dependencies. It added unnecessary dependencies and it is missing quite a few of them necessary to build the package</span> |
+| **Run Requirements**  | <span style="color: green;">Missing just ``setuptools`` from the host requirements (but this dependency is not defined on ``pytest`` package)</span> | <span style="color: red;">Incorrectly dependencies added to the project, it will not build the recipe as well</span>  |
+| **Selectors** | <span style="color: green;">Skipping correctly Python 2 and added selectors for windows and python versions</span> | <span style="color: red;">it didn't add any information regarding selectors</span> |
```

Comment:
[The comment](https://github.com/conda-forge/blog/pull/9/files#r389644319) about the bad sys.platform selectors should go here.